voicebox/node.py

```diff
@@ -150,5 +199,31 @@ def end_call(self, addr: str, inform_connection: bool = True):
     @staticmethod
     def validate_connection(address: str) -> bool:
 
-        return bool(input("Would you like to connect to this client at {}? ".format(address)))
+        return True
+
+        # return bool(input("Would you like to connect to this client at {}? ".format(address)))
+
+    @property
+    def encryption_pipeline(self) -> List[BaseEncryptor]:
+
+        return self.__encryption_pipeline
+
+    @encryption_pipeline.setter
+    def encryption_pipeline(
+        self,
+        encryption_pipeline: List[BaseEncryptor]
+    ):
+        for encryptor in encryption_pipeline:
+
+            self.append_to_encryption_pipeline(encryptor)
+
+    def append_to_encryption_pipeline(self, encryptor: BaseEncryptor):
+
+        if not isinstance(encryptor, type(BaseEncryptor)):
+
+            raise ValueError(

```

Comment:
The `append_to_encryption_pipeline` method raises a `ValueError` if the `encryptor` does not extend from `BaseEncryptor`. However, the error message is not very descriptive. Consider improving the error message to provide more context, such as "The provided encryptor class {encryptor.__name__} does not extend from BaseEncryptor". [medium]